libraries/lib-music-information-retrieval/StftFrameProvider.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <vector>
+
+namespace MIR
+{
+class MirAudioReader;
+
+class StftFrameProvider
```

Comment:
Clarify with comments that this class does not calculate STFT, but is only a source of time-domain frames